main.cpp

```diff
@@ -278,12 +278,13 @@ int main (int argc, char** argv)
 				std::cerr << "Error dispatching: " << retval << std::endl;
 			else if (retval != 0)
 				packets_read = true;
+			else
```

Comment:
So I am afraid there would be some bug due to the the following code

	int process_tcp (u_char * userdata, const dp_header * header, const u_char * m_packet) {
	struct dpargs * args = (struct dpargs *) userdata;
	struct tcphdr * tcp = (struct tcphdr *) m_packet;

	curtime = header->ts;